headers/gamepad.h

```diff
@@ -188,6 +188,8 @@ class Gamepad {
 	GamepadButtonMapping *mapAnalogRSXPos;
 	GamepadButtonMapping *mapAnalogRSYNeg;
 	GamepadButtonMapping *mapAnalogRSYPos;
+	GamepadButtonMapping *map4WayMode;
```

Comment:
Take a look at the enum "SUSTAIN_FOCUS_MODE" for how we want to do this.

What we want is a single GpioAction pin definition that allows a user to toggle their current 4-8-way mode to the other setting. So basically its a boolen-not that toggles whatever setting you have assigned to gamepad.